### ISSUE TITLE
xcube server error: No module named 'jwt'

### ISSUE THREAD

**Issue by joaofelipecsantos** [dcs4cop/xcube#291]
**Describe the bug**  
Today (24/03/2020) I updated the xcube, xcube-gen-bc and xcube-viewer.  
  
I created the cube in zarr format using the xcube gen.  
I am trying to run the xcube serve now and I am receiving an internal error message: No module 'jwt'.  
I don't know what I am doing wrong.  
  
![image](https://user-images.githubusercontent.com/32440152/77450716-f863ec00-6df3-11ea-8dc7-f9c225586583.png)  
  
**To Reproduce**
1. Generate the cube using the code below:  
xcube gen --config <gen_config_full_path.yml> <input_dir_full_path/*.nc>
The cube was generated as supposed.  
  
2. Try to run the xcube serve using the code below:  
xcube serve --verbose --traceperf --config <serve_config_full_path.yml>  
  
3. Inside configuration xcube serve .yml file the Datasets are okay, the Styles are okay.

4. The error is in the image above.  
  
**Expected behavior**  
Expect to receive message that the cube was successfully added to local host.  
  
**Screenshots**  
Figure above.  
  
**Desktop (please complete the following information):**  
 - OS: Linux Ubuntu 18.04

**Comment by AliceBalfanz** [comment]
Hi Joao, 
thanks for your message, please try the following:
1. go to your xcube repository
2. activate the xcube envirionment (`conda activate xcube`)
3. next update the environment with: `conda env update -n xcube --file environment.yml  --prune`
4. run the python setup.py script `python setup.py develop`

You might have to update yarn for the viewer as well:
1. go to your xcube-viewer repository
2. execute `yarn install`

Please let me know, if this was the solution to the occurring issue :)
Best, 
Alicja

**Comment by joaofelipecsantos** [comment]
Dear,  
  
I would like to to update you with my xcube status.  
  
I installed manually the modules "jwt", "pyjwt", "moto" inside xcube environment and now I don't have more the error to run "xcube serve".  
  
I had the same error when I tried to install in a new machine. So, I don't know why on my computers I need to install these modules manually.  
  
But now is everything working fine.  
Regards,  
João Santos.
